### ISSUE TITLE
Master is broken with a UnicodeDecodeError

### ISSUE THREAD

**Issue by ocefpaf** [conda-forge/staged-recipes#3152]
Not sure what/which file is causing this but we need a more verbose output to help debug this.

```
Initialized empty Git repository in /private/var/folders/vy/rcv48w3j4w79llzf_x6qnvw40000gn/T/tmpV5YLGF__feedstocks/libgdal-feedstock/.git/
Traceback (most recent call last):
  File "/Users/travis/miniconda/bin/conda-smithy", line 6, in <module>
    sys.exit(conda_smithy.cli.main())
  File "/Users/travis/miniconda/lib/python2.7/site-packages/conda_smithy/cli.py", line 283, in main
    args.subcommand_func(args)
  File "/Users/travis/miniconda/lib/python2.7/site-packages/conda_smithy/cli.py", line 100, in __call__
    generate_feedstock_content(feedstock_directory, args.recipe_directory, meta)
  File "/Users/travis/miniconda/lib/python2.7/site-packages/conda_smithy/cli.py", line 31, in generate_feedstock_content
    configure_feedstock.copytree(source_recipe_dir, target_recipe_dir)
  File "/Users/travis/miniconda/lib/python2.7/site-packages/conda_smithy/configure_feedstock.py", line 580, in copytree
    copytree(s, d, ignore, root_dst=root_dst)
  File "/Users/travis/miniconda/lib/python2.7/site-packages/conda_smithy/configure_feedstock.py", line 582, in copytree
    copy_file(s, d)
  File "/Users/travis/miniconda/lib/python2.7/site-packages/conda_smithy/feedstock_io.py", line 80, in copy_file
    for line in fh_src:
  File "/Users/travis/miniconda/lib/python2.7/codecs.py", line 314, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xfb in position 0: invalid start byte
Traceback (most recent call last):
  File ".CI/create_feedstocks.py", line 152, in <module>
    '--feedstock-directory', feedstock_dir])
  File "/Users/travis/miniconda/lib/python2.7/subprocess.py", line 186, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['conda', 'smithy', 'init', '/Users/travis/build/conda-forge/staged-recipes/recipes/libgdal', '--feedstock-directory', 
```

**Comment by isuruf** [comment]
This is probably one of the binary files in the libgdal recipe

**Comment by ocefpaf** [comment]
Yep. I am trying to work around it. PR soon.

**Comment by ocefpaf** [comment]
This is "fixed" with the workaround in #3153 

But a long term fix is under way too in https://github.com/conda-forge/conda-smithy/pull/535
